<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>539</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Изменение параметров режима</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройка режима:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Тестовый</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>641</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры возмущения</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;мощность возмущающего воздействия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Мощность возмущения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>МВт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;место возмущения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Узел возмущения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Г1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Г2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>НГ1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>НГ2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>НГ3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>НГ4</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отключаемя линия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Откл. линия</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Line_off">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Г1 - Н1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Н1 - Н2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Н2 - Н3 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Н3 - Г2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Г2 - Н4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Н4 - Г1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Марка провода выбирается для всех линий в системе (определяет пропускаему мощность в режиме одной связи)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Марка провода</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Line_mark">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>АС-120/19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>АС-240/32</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="Line_fault">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключение ЛЭП</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="knot_fault">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Возм. в узле нагрузки или генерации</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>641</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Уставки АЧР 1 / АЧР 2</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выделенная мощность на АЧР 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Мощность P, МВТ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R2_P_field">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Уставка очередей по времени&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Время срабатвывания, с</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R1_f_field">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>АЧР 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Частота срабатывания между очередями АЧР&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Частота ступеней срабатывания, Гц</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R1_t_field">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>АЧР 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R2_t_field">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выделенная мощность на АЧР2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Мощность P, МВТ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R1_P_field">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>35</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время срабатвывания, с</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Уставка очередей по времени&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Число очередей</string>
    </property>
   </widget>
   <widget class="QComboBox" name="A4R1_count_stpes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Уставка очередей по времени&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Число очередей</string>
    </property>
   </widget>
   <widget class="QComboBox" name="A4R2_count_stpes">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Уставка возврата, Гц</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="A4R2_vozvr_field">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Можно изменять мощности нагрузок прямо во время симуляции&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="S_basis_field">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>400</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_basis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Базисная мощность&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Sб</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_basis_izm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>49</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>МВА</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Построит дополнительные графики по углу передачи и мощности генераторов. В отдельном окне в конце симуляции&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Дополнительные построения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Момент возмущаего воздейсвтия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Время возмущения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>с</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="h_field">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0.02</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;шаг симуляции&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>сек</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Pvozvr_check">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Построит дополнительные графики по углу передачи и мощности генераторов. В отдельном окне в конце симуляции&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Возврат мощности</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Момент возмущаего воздейсвтия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Время возврата мощности</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pvozvr">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>с</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
